test/System/Vector2.cpp

```diff
@@ -184,11 +189,97 @@ TEST_CASE("sf::Vector2 class template - [system]")
         }
     }
 
+    SUBCASE("Length and normalization")
+    {
+        sf::Vector2f v(2.4f, 3.0f);
+
+        CHECK(v.length() == Approx(3.84187));
+        CHECK(v.lengthSq() == Approx(14.7599650969));
+        CHECK(v.normalized() == ApproxVec(0.624695, 0.780869));
```

Comment:
I've made a point to mark such local variables as `const` in tests because it allows us to test that all the functions we're calling are marked as `const` as well. This fact is obvious from reading the code but I think it's worthwhile to maintain const-correctness in tests since certain member functions do make distinctions between being `const` or non-`const`. This can be fixed later since there are other tests where I'd like to add `const` to local variables.